src/chrome/content/rules/Grubstreet.com.xml

```diff
@@ -6,13 +6,33 @@
 
 	Redirected:
 		client.grubstreet.com
+		chicago.grubstreet.com
+		losangeles.grubstreet.com
+		newyork.grubstreet.com
+		philadelphia.grubstreet.com
+		sanfrancisco.grubstreet.com
 
 	Refused:
 		grubstreet.com (non-www)
+		boston.grubstreet.com
+		chicago.grubstreet.com
+		losangeles.grubstreet.com
+		newyork.grubstreet.com
+		philadelphia.grubstreet.com
+		sanfrancisco.grubstreet.com
 -->
 <ruleset name="grubstreet.com">
 	<target host="www.grubstreet.com" />
 
+	<rule from="^http://boston\.grubstreet\.com/" to="https://www.grubstreet.com/" />
+	<rule from="^http://chicago\.grubstreet\.com/" to="https://www.grubstreet.com/" />
+	<rule from="^http://losangeles\.grubstreet\.com/" to="https://www.grubstreet.com/" />
+	<rule from="^http://newyork\.grubstreet\.com/" to="https://www.grubstreet.com/" />
+	<rule from="^http://philadelphia\.grubstreet\.com/" to="https://www.grubstreet.com/" />
+	<rule from="^http://sanfrancisco\.grubstreet\.com/" to="https://www.grubstreet.com/" />
+
+	<rule from="^http://grubstreet\.com/" to="https://www.grubstreet.com/" />
```

Comment:
You also need to add them as target

Reply:
Actually is needed if they're redirecting them to www  anyway?

Reply:
You can only redirect `from` `target host`s